app/src/main/java/com/framgia/soundclound/data/source/repository/GenresRepository.java

```diff
@@ -0,0 +1,52 @@
+package com.framgia.soundclound.data.source.repository;
+
+import android.support.annotation.StringDef;
+
+import com.framgia.soundclound.R;
+import com.framgia.soundclound.data.model.Genres;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.framgia.soundclound.data.source.repository.GenresRepository.ServiceName.*;
+
+/**
+ * Created by Bui Danh Nam on 5/1/2018.
+ */
+
+public class GenresRepository {
+    @Retention(RetentionPolicy.SOURCE)
+    @StringDef({
+            GENRES_ALL, GENRES_POP, GENRES_CLASSIC,
+            GENRES_POP_BLAS, GENRES_DEEP_HOUSE, GENRES_COUNTRY,
+            GENRES_ROCK, GENRES_EDM, GENRES_ELECTRONIC
+    })
+    public @interface ServiceName {
+        String GENRES_ALL = "all-music";
+        String GENRES_POP = "pop";
+        String GENRES_COUNTRY = "country";
+        String GENRES_POP_BLAS = "jazzblues";
+        String GENRES_DEEP_HOUSE = "deephouse";
+        String GENRES_ROCK = "rock";
+        String GENRES_EDM = "danceedm";
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.